### ISSUE TITLE
WORK IN PROGRESS []: ARIA progressbar nodes must have an accessible name

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#17342]
- [508-defect-0 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-0)
    - [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)
    - [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)
    - [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)
    - [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** _First name only. Delete if not applicable._

**Comment by joshkimux** [comment]
@Mottie this should be addressed by @1Copenut 's [bug report here](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/357#issuecomment-743478883)

**Comment by 1Copenut** [comment]
@joshkimux Do you have a preferred way to track these sitewide issues in instances like this form? In the past, I've kept the instance tickets open, and referenced the sitewide issue in the description so there's a link between the two. But that's just one way; feel free to handle them as you see fit for your program!

**Comment by joshkimux** [comment]
@1Copenut Was not aware of that! I'll reopen this so we can keep track of it! Thank you for the context.
